lib/pre_assembly/remediation/remediate.rb

```diff
@@ -59,38 +56,32 @@ def get_object
       # remediate the object using versioning or not depending on its current state;
       # returns true or false depending on if all steps were successful
       def remediate
-        begin
-          DruidTools::Druid.new(@pid) # this will confirm the druid is valid, if not, we get an exception which is caught below and logged
-          get_object
-
-          if should_remediate? # should_remediate? == true
-
-            if updates_allowed?
-              if versioning_required?
-                update_object_with_versioning
-                @message = 'remediated with versioning' if @success
-              else
-                update_object
-                @message = 'remediated directly (versioning not required)' if @success
-              end
+        DruidTools::Druid.new(@pid) # this will confirm the druid is valid, if not, we get an exception which is caught below and logged
+        get_object
+
+        if should_remediate? # should_remediate? == true
+          if updates_allowed?
+            if versioning_required?
+              update_object_with_versioning
+              @message = 'remediated with versioning' if @success
             else
-              @success = false
-              @message = 'currently in accessioning, cannot remediate'
+              update_object
+              @message = 'remediated directly (versioning not required)' if @success
             end
-
-          else # should_remediate? == false, no remediation required
-
-            @success = true
-            @message = 'remediation not needed'
-
-           end
-        rescue Exception => e
-          @success = false
-          @message = "#{e.message}"
-          Honeybadger.notify(e)
+          else
+            @success = false
+            @message = 'currently in accessioning, cannot remediate'
+          end
+        else # should_remediate? == false, no remediation required
+          @success = true
+          @message = 'remediation not needed'
         end
-
-        @success
+      rescue Exception => e
+        @success = false
+        @message = "#{e.message}"
+        Honeybadger.notify(e)
+      ensure
+        return @success
```

Comment:
@atz, i have a dumb ruby question:  is the `return` strictly necessary here?  not asking to change it if it isn't (i actually find explicit returns more readable and prefer them in a lot of cases, this one included).  just curious.